contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -321,5 +321,44 @@ def testTraverseInstancedScene ( self ) :
 		self.assertTrue( isinstance( instance0Object, IECoreScene.SpherePrimitive ) )
 		self.assertTrue( isinstance( instance1Object, IECoreScene.SpherePrimitive ) )
 
+	def testGeometricInterpretation( self ) :
+
+		primitive = IECoreScene.PointsPrimitive( IECore.V3fVectorData( [ imath.V3f( 0 ) ] ) )
+		self.assertEqual( primitive["P"].data.getInterpretation(), IECore.GeometricData.Interpretation.Point )
+
+		for data, interpolation in [
+			( IECore.V3fData( imath.V3f( 0 ) ), IECoreScene.PrimitiveVariable.Interpolation.Constant ),
+			( IECore.V3fVectorData( [ imath.V3f( 0 ) ] ), IECoreScene.PrimitiveVariable.Interpolation.Vertex ),
+		] :
+
+			for interpretation in [
+				IECore.GeometricData.Interpretation.Point,
+				IECore.GeometricData.Interpretation.Vector,
+				IECore.GeometricData.Interpretation.Normal,
+				IECore.GeometricData.Interpretation.None_,
+			] :
+
+				dataCopy = data.copy()
+				dataCopy.setInterpretation( interpretation )
+
+				name = dataCopy.typeName() + str( interpretation )
+				primitive[name] = IECoreScene.PrimitiveVariable( interpolation, dataCopy )
+
+		root = IECoreScene.SceneInterface.create( "/tmp/interpretationTest.usda", IECore.IndexedIO.OpenMode.Write )
```

Comment:
How come these other ones aren't causing problems?

```
> grep -r /tmp test/IECore*/*.py | wc -l
150
```

Reply:
I just checked and its because our internal CI is disabled for Cortex! Possibly because it was regularly failing, potentially with issues just like this... Yikes...